pisa/stages/osc/prob3numba/numba_osc.py

```diff
@@ -115,27 +120,30 @@ def get_H_mat(rho, nsi_eps, nubar, H_mat):
     # 2*sqrt(2)*Gfermi in (eV^2-cm^3)/(mole-GeV)
     tworttwoGf = 1.52588e-4
     a = 0.5 * rho * tworttwoGf
-    if nubar == -1:
-        a = -a
 
     # standard matter interaction Hamiltonian
     clear_matrix(H_mat)
-    H_mat[0,0] = a
+    H_mat[0, 0] = 0 
```

Comment:
I believe `clear_matrix()` sets all elements of a matrix to 0, so this should be unnecessary, I think...